aws/resource_aws_secretsmanager_secret_version.go

```diff
@@ -21,6 +21,10 @@ func resourceAwsSecretsManagerSecretVersion() *schema.Resource {
 		},
 
 		Schema: map[string]*schema.Schema{
+			"arn": &schema.Schema{
```

Comment:
Nit: the `&schema.Schema` here is extraneous since Go 1.7 as its previously defined by the `map[string]*schema.Schema` above